run.py

```diff
@@ -1,8 +1,10 @@
 from backend.app import create_app
 from config import config
+from backend.routes.auth import auth_bp
 import os
 
 if __name__ == '__main__':
     config_name = os.environ.get('FLASK_ENV') or 'default'
     app = create_app(config[config_name])
-    app.run(host='0.0.0.0', port=4000, debug=True)
+    app.run(host='0.0.0.0', port=7000, debug=True)
```

Comment:
@Archiljain Port configuration changes is out of scope.. you can switch port on your local via .env file. please see the readme file.